pkg/resource/testdata/TestStatefulSetBuilder/automount_sa.golden

```diff
@@ -31,7 +31,7 @@ spec:
         - /bin/bash
         - -ecx
         - 'exec /cockroach/cockroach.sh start --advertise-host=$(POD_NAME).test-cluster.test-ns
-          --certs-dir=/cockroach/cockroach-certs/ --http-port=8080 --sql-addr=:26257
+          --certs-dir=/cockroach/cockroach-certs/ --http-addr=:8080 --sql-addr=:26257
           --listen-addr=:26258 --log="{sinks: {stderr: {channels: [OPS, HEALTH], redact:
```

Comment:
Wait, did we already make the change to `listen-addr` in the operator? What value is it pulling from?

Reply:
Yes we already have change of listen-addr which is equal to `:grpcPort`, I did the same thing from http-port to http-addr=`:httpPort`